yaml/parse_test.ts

```diff
@@ -240,3 +240,31 @@ Deno.test({
     });
   },
 });
+
+Deno.test({
+  name: "parse() handles omap type",
+  fn() {
+    const yaml = `--- !!omap
+- Mark McGwire: 65
+- Sammy Sosa: 63
+- Ken Griffey: 58
+`;
+    assertEquals(parse(yaml), [
+      { "Mark McGwire": 65 },
+      { "Sammy Sosa": 63 },
+      { "Ken Griffey": 58 },
+    ]);
+
+    // Invalid omap
+    // map entry is not an object
+    assertThrows(() => parse("--- !!omap\n"));
```

Comment:
Can you please add the error class and message arguments for `assertThrows()`? Ditto for the other instances.